packages/api-gateway/src/drivers/http/adapters/users/updateUser.js

```diff
@@ -0,0 +1,92 @@
+const errorHandler = require('./errorHandler');
+
+async function updateUser(req, reply) {
+  try {
+    const {
+      email,
+      avatar,
+      firstName,
+      secondName,
+      firstSurname,
+      secondSurname,
+      birthDate,
+      nationality,
+      dniId,
+      dniFrontImg,
+      dniBackImg,
+      gender,
+      phoneNumber,
+      emergencyNumber,
+      passport,
+      address: {
+        country, city, state, address, zip,
+      },
+    } = req.body;
+    const { userId } = req.params;
+
+    req.log.info('[http-server]: Creating user with: ', {
```

Comment:
Instead of passing the all the params why not create a function that return an object with all the info that should be logged, for example:

````js
function buildUseInfoToLog (userInfo) {
  return {
    user: user.name,
    //...
  }
}

req.log.info('[http-server]: Creating user with: ', buildUserInfoLog(userInfo));
````

Looks clean right? What do you think?